src/pages/Main/components/Technologies/Technologies.module.scss

```diff
@@ -13,14 +29,47 @@
         width: 32.7%;
         max-height: 24.0625rem;
         margin: 0.375rem 0;
+        overflow: hidden;
+        transition: .3s;
+        &_Big {
+            display: flex;
+            position: relative;
+            width: 49.523%;
+            max-height: 24.0625rem;
+            margin: 0.375rem 0;
+            overflow: hidden;
+            transition: .3s; 
+            &:hover {
+                .technoPic__title {
+                    right: 0;
+                    transition: .5s;
+                }
+            }              
+        }
+        &:hover {
+            .technoPic__title {
+                right: 0;
+                transition: .5s;
+            }
+        }
     }
-    &__ContBig {
+    &__title {
         display: flex;
-        position: relative;
-        width: 49.523%;
-        max-height: 24.0625rem;
-        margin: 0.375rem 0;
+        position: absolute;
+        width: 17.25rem;
+        height: 3.875rem;
```

Comment:
is too little (espesially in width) for the screen resolution less then 768px, relatively to parent element (must be 225px in resolution 375px, and now it is 130px)